### DISCUSSION TITLE
db file flocked by another, or taking too long

### DISCUSSION THREAD

**Discussion by smallemployees** [etcd-io/etcd#18125] [Q&A]
### Bug report criteria

- [ ] This bug report is not security related, security issues should be disclosed privately via [etcd maintainers](mailto:etcd-maintainers@googlegroups.com).
- [ ] This is not a support request or question, support requests or questions should be raised in the etcd [discussion forums](https://github.com/etcd-io/etcd/discussions).
- [ ] You have read the etcd [bug reporting guidelines](https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/reporting_bugs.md).
- [ ] Existing open issues along with etcd [frequently asked questions](https://etcd.io/docs/latest/faq) have been checked and this is not a duplicate.

### What happened?

Etcd node failed to start

### What did you expect to happen?

How can I release this lock when the underlying layer is NFS storage

### How can we reproduce it (as minimally and precisely as possible)?

How can I release this lock when the underlying layer is NFS storage

### Etcd version (please run commands below)

<details>

```console
$ etcd --version
# paste output here

$ etcdctl version
# paste output here
```3.4.9

</details>


### Etcd configuration (command line flags or environment variables)

<details>

# paste your configuration here

</details>


### Etcd debug information (please run commands below, feel free to obfuscate the IP address or FQDN in the output)

<details>

```console
$ etcdctl member list -w table
# paste output here

$ etcdctl --endpoints=<member list> endpoint status -w table
# paste output here
```

</details>

**Comment by jmhbnz** [comment]
Hi @smallemployees - Thanks for your question. Can you please confirm which version of etcd server you are running?
